ext/AvizExt/viz/clustering.jl

```diff
@@ -92,18 +91,19 @@ function _plot_clusters_confint!(
     legend_entry = Vector{Any}(undef, length(unique_clusters))
 
     x_timesteps::UnitRange{Int64} = 1:length(timesteps(data))
+    slice_dimension = filter(x -> x != :timesteps, dimnames(data))[1]

```

Comment:
Is there some consideration that I can't see here?

Seems to me `dimnames(data) .!= :timesteps` should work?

`filter()` is comparatively expensive to use, which is why I'm raising this.

Reply:
`dimnames(data) .!= :timesteps` returns a `Bitvector`, but what I want is the name of the dimension that is not `:timesteps` (so, a `Symbol`). That said, what I can do is:

```julia
dimnames(data)[findfirst(dimnames(data) .!= :timesteps)]
```

or

```julia
data_dims = collect(dimnames(data))
data_dims [data_dims  .!= :timesteps)][1]
```

Any other ideas..? (unfortunately I have to convert `dimnames(data)` before filtering as it doesn't work with `Tuples`...)

Reply:
`dimnames(data)[findfirst(dimnames(data)`

This is what I'm thinking of then, except storing `dimnames` first sonewhere